prow/plugins.yaml

```diff
@@ -235,6 +235,30 @@ repo_milestone:
     maintainers_id: 3169231
     maintainers_team: community-milestone-maintainers
     maintainers_friendly_name: Community Milestone Maintainers
+  kubernetes/website:
+    maintainers_id: 2953475
+    maintainers_team: sig-docs-en-owners
```

Comment:
Once https://github.com/kubernetes/org/pull/628 gets merged, you can add only the `website-milestone-maintainers` team here :smile: